packages/ember-views/lib/views/text_field.js

```diff
@@ -54,6 +54,8 @@ export default Component.extend(TextSupport, {
     'width'
   ],
 
+  defaultLayout: null,
```

Comment:
Need some feedback here. text fields are input tags. In IE8 an exception is raised if you try to append blank text nodes to an input.

text_field is now a component, so a layout makes sense. Not sure why it is being appended to the view element though.